### PR TITLE
Security section to the website

### PR DESCRIPTION
**Pull request by abstractj**
@stianst I believe this makes the way to report security vulnerabilities more evident. Some considerations:

1. I removed the section mentioned the way to create Jiras with the security sensitive field, because I believe it makes the whole process over complicated. People should be able to just send an e-mail and report it.

2. Would be nice to include an ETA for people to get some response. This gives people some confidence that the problem is being handled. Maybe something like:
```
The Keycloak team will acknowledge your e-mail in 24 hours, and you will receive a response indicating the next steps in handling your report

After the initial reply to your e-mail, the team will keep you informed about the progress being made towards a fix, and may ask for additional information or guidance surrounding the reported issue. These updates will be sent at least every X days.
```

### CODE REVIEW

**Comment by stianst** [comment]
On 05/10/17 04:24, Bruno Oliveira da Silva wrote:
>
> @Stian <https://github.com/stian> I believe this makes the way to 
> report security vulnerabilities more evident. Some considerations:
>
> 1.
>
>     I removed the section mentioned the way to create Jiras with the
>     security sensitive field, because I believe it makes the whole
>     process over complicated. People should be able to just send an
>     e-mail and report it.
>

We should include the JIRA process as well as that is important. 
Otherwise we would have to explain this to everyone after they've sent 
the email. Or we would have to do it and then figure out the guys JIRA 
nick so we could give them visibility.

> 1.
>
>
> 2.
>
>     Would be nice to include an ETA for people to get some response.
>     This gives people some confidence that the problem is being
>     handled. Maybe something like:
>
> |The Keycloak team will acknowledge your e-mail in 24 hours, and you 
> will receive a response indicating the next steps in handling your 
> report After the initial reply to your e-mail, the team will keep you 
> informed about the progress being made towards a fix, and may ask for 
> additional information or guidance surrounding the reported issue. 
> These updates will be sent at least every X days.|

We're a community project and do not have resources to respond within 24 
hours. We are also unable to fix quickly in community as security 
vulnerabilities will most likely be embargoed and require pushing out to 
customers first.

**Comment by abstractj** [comment]
@stianst updated!

File: community.html [hunk 1]
```diff
@@ -90,28 +92,6 @@
             and send us an email. We'll typically answer within a few days (excluding weekends of course).
         </p>
 
-        <h3>Reporting security vulnerabilities</h3>
```
Comment (stianst) [inline community.html]: I'd leave the heading and add a brief text that links to the security page.

File: about.html [hunk 2]
```diff
@@ -66,6 +66,7 @@
                     <li><a href="documentation.html">Docs</a></li>
                     <li><a href="downloads.html">Downloads</a></li>
                     <li><a href="community.html">Community</a></li>
+                    <li><a href="security.html">Security</a></li>
```
Comment (stianst) [inline about.html:69]: Shouldn't we remove the "Source" entry? Isn't the menu to wide otherwise?

**Review by stianst — COMMENTED** [review]
Looks good, thanks for doing this. Two small comments.

**Pull request merged**